app/server/app/public/widgets/communityWidget.html

```diff
@@ -49,15 +49,18 @@
         height: 100%;
         font-family: Verdana, Geneva, sans-serif;
         font-size: 13px;
-        background-image: url(http://localhost:9090/widgets/boating.jpg);
+        background-image: url(/widgets/boating.jpg);
         background-position: center;
         background-size: cover;
       }
 
       .widget-content {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-around;
         padding: 10px;
         border-radius: 7px;
-        height: calc(100% - 20px);
+        height: calc(100% - 40px);
```

Comment:
@cschwinderg the height needs to be the viewport height minus the padding – otherwise the overlay doesn't cover the bottom of the widget. I also don't think equally distributing the content (sentence and form field) as the height increases works very well visually – it looks better to keep everything aligned at the top.